magento/pwa-studio
# [Doc] Add note about Node 12 deprecation warnings

## Description

Adds a note about a deprecation warning that shows up in the log when using Node12. This is caused by a dependency and is noted where we specify Node requirements.

## Related Issue





Closes PWA-698

## Acceptance 



Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps


1. Run HTML preview server
1. Verify note appears on the following pages:

    -  `/tutorials/pwa-studio-fundamentals/`
    - `/tutorials/pwa-studio-fundamentals/project-setup/`
    - `/venia-pwa-concept/features/modular-components/`
    - `/venia-pwa-concept/setup/`

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
